packages/hd-cache/src/index.ts

```diff
@@ -485,17 +487,21 @@ export function publicKeyToMultisigArgs(publicKey: HexString): HexString {
   const M = 1;
   const publicKeyHashes = [blake160];
 
-  const serialized =
-    "0x00" +
-    ("00" + R.toString(16)).slice(-2) +
-    ("00" + M.toString(16)).slice(-2) +
-    ("00" + publicKeyHashes.length.toString(16)).slice(-2) +
-    publicKeyHashes.map((h) => h.slice(2)).join("");
+  const serialized = bytes.concat(
+    [0],
+    Uint8.pack(R),
+    Uint8.pack(M),
+    Uint8.pack(publicKeyHashes.length),
+    ...publicKeyHashes
+  );
 
+  // hash160
+  // 0x prefix + 20 bytes hex = 2 + (20 * 2) = 42
+  const hash160HexLength = 42;
   const args = new utils.CKBHasher()
     .update(serialized)
     .digestHex()
-    .slice(0, 42);
+    .slice(0, hash160HexLength);
   return args;
```

Comment:
These are optional
```ts
return utils.ckbHash(serialized).slice(0, hash160HexLength);
```
In another PR, adding a `type?: 'blade160'` param to `ckbHash()` 
```ts
function ckbHash(data: BytesLike, type?: 'blade160'): Hash {
  const LENGTHS = {
    blade160: 42
  }
  const hasher = new CKBHasher();
  hasher.update(bytes.bytify(data));
  return hasher.digestHex(0, LENGTHS[type] ?? Infinity);
}
```
After
```ts
return utils.ckbHash(serialized, 'blade160')
```

Reply:
Good point. It would be better if Lumos provided a function for `blake160` since `hash160` is common in CKB. As `CKBHasher` follows a general `Hasher` interface from NodeJS, a new function may be more suitable.

Reply:
I've added a new API `ckbHash160` to the `utils`. @twhy can you please review and check if the comment can be resolved

Reply:
Sure. But it seems there's no `Resolve` button for comment.